### PR TITLE
Add OpenDyslexic

### PR DESCRIPTION
**Pull request by kirbyfan64**
This is my first addition to jsdelivr, so please tell me if I did something wrong. :)

### CODE REVIEW

**Comment by TheTannerRyan** [comment]
The github page referenced in the info.ini leads to a 404. I have also submitted OpenDyslexic to jsDelivr with auto update linked my repo so I can auto update the fonts on the CDN when Abelardo updates them.

**Comment by kirbyfan64** [comment]
@TheTannerRyan Wouldn't it work when/if this gets merged?

**Comment by TheTannerRyan** [comment]
I don't know if they would merge it, if it is leading to a 404. It would work, but it depends on who is merging it, if they would allow it. Also, every time the font gets updated, you will have to fork, commit and send a pull request again. With auto update, all you have to do is update the repo. A bot will check and see that it has changed and update it. If you want you can close this. I have already submitted one that has auto update, and passed the auto tests.

File: files/open-dyslexic/info.ini [hunk 1]
```diff
@@ -0,0 +1,4 @@
+author = "Abelardo Gonzalez"
+github = "https://github.com/mrkindy/jsdelivr/files/open-dyslexic"
```
Comment (megawac) [inline files/open-dyslexic/info.ini]: Usually this should point to a repository - this isn't good enough in this case. If theres a website we could substitute it for an empty string

Comment (megawac) [inline files/open-dyslexic/info.ini]: In this case https://github.com/antijingoist/open-dyslexic

Comment (kirbyfan64) [inline files/open-dyslexic/info.ini]: @megawac Fixed.

**Comment by kirbyfan64** [comment]
Looks like the bot likes me now!

**Comment by TheTannerRyan** [comment]
One more thing. The latest version is 2.1.0 not 3.0. And you will have to squash all the commits into one.

**Comment by kirbyfan64** [comment]
No; I uploaded version 3.0. It's newer. I converted the files from ttf to the other formats myself. I should know; antijingoist is my brother.

I'll squash them in a few.

**Comment by kirbyfan64** [comment]
@TheTannerRyan Squashed.

**Comment by megawac** [comment]
Alright just need to fix the version and looks good to go

See https://github.com/antijingoist/open-dyslexic/blob/master/bower.json#L5

**Comment by jimaek** [comment]
Looks like this PR was moved here https://github.com/jsdelivr/jsdelivr/pull/3962

**Comment by kirbyfan64** [comment]
@megawac @jimaek THE VERSION IS CORRECT! The bower script says 2.1, but I used the new version, 3.0, a major redesign.

Also, 3.0 uses a new, more permissive license.

Please reopen.

Actually, you know what, pinging @antijingoist.

**Comment by jimaek** [comment]
I believe you, all I need is some official documentation showing that.

**Comment by kirbyfan64** [comment]
Proof?

Compare [the official TTF for version 3.0](https://github.com/antijingoist/open-dyslexic/blob/master/ttf/OpenDyslexic3-Regular.ttf) with the TTF's I uploaded.

You can also go to the OpenDyslexic website. The Download page mentions version 3. You can compare the downloads there with my TTF files.

**Comment by jimaek** [comment]
Thats the problem that me and megawac are having. A `3` in a filename and a vague mention on the website is not enough. 
The author should tag the 3.0 version on github, or mention it on the README, or make a clearly titled "v3.0" download link available. Searching for clues everywhere to find the latest version doesnt look very well. I would more likely assume its some beta version not intended for production.

I think that the requirements are reasonable.

**Comment by antijingoist** [comment]
The assumption is correct, which is why the versioning has not been updated. 

I will be updating both in a few weeks  , but it looks like the any appropriate course of action were has been taken?

**Comment by kirbyfan64** [comment]
So...would it be possible for *both* versions to be available, with 2.1 as the default?

**Comment by jimaek** [comment]
@antijingoist Thank you for clearing everything for us :)
@kirbyfan64 Sure, but not until 3.0 is stable. I will merge the 2.1 PR. Once 3.0 is official we can add it as well.

**Pull request closed without merging**